packages/email/mailer.ts

```diff
@@ -46,10 +46,14 @@ const getTransport = () => {
     host: process.env.NEXT_PRIVATE_SMTP_HOST ?? 'localhost:2500',
     port: Number(process.env.NEXT_PRIVATE_SMTP_PORT) || 587,
     secure: process.env.NEXT_PRIVATE_SMTP_SECURE === 'true',
-    auth: {
-      user: process.env.NEXT_PRIVATE_SMTP_USERNAME ?? '',
-      pass: process.env.NEXT_PRIVATE_SMTP_PASSWORD ?? '',
-    },
+    ignoreTLS: process.env.NEXT_PRIVATE_SMTP_SECURE === 'false' ? true : undefined,
```

Comment:
This is probably best served under a new env var:

`NEXT_PRIVATE_SMTP_UNSAFE_IGNORE_TLS`

Since skipping startls may be something you want but straight up ignoring bad certs feels like yet another specific usecase.

Reply:
done